presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -144,6 +144,8 @@
     private static final String HUDI_METADATA_ENABLED = "hudi_metadata_enabled";
     private static final String READ_TABLE_CONSTRAINTS = "read_table_constraints";
     public static final String READ_MASKED_VALUE_ENABLED = "read_null_masked_parquet_encrypted_value_enabled";
+    public static final String INSERT_NON_MANAGED_TABLE = "insert_non_managed_table";
```

Comment:
nit: How about naming the variable as an adj such as `INSERT_NON_MANAGED_TABLE_ENABLED` instead of a noun?